templates/mysqlbackup.sh.erb

```diff
@@ -27,7 +31,9 @@ PATH=<%= @execpath %>
 
 
 
+<%- if @kernel == 'Linux' -%>
 set -o pipefail
```

Comment:
wondering if we can make the shell dependent on whether `/bin/bash` actually exists… but then again, FreeBSD puts bash in `/usr/local/bin` (and doesn't install it per default)